api/src/test/scala/hmda/api/http/InstitutionsHttpApiSpec.scala

```diff
@@ -73,9 +74,11 @@ class InstitutionsHttpApiSpec extends WordSpec with MustMatchers with ScalatestR
       }
       Get("/institutions/12345/filings/xxxx") ~> institutionsRoutes ~> check {
         status mustBe StatusCodes.NotFound
+        responseAs[ErrorResponse] mustBe ErrorResponse(404, "xxxx filing not found for institution 12345", "institutions/12345/filings/xxxx")
       }
       Get("/institutions/xxxxx/filings/2017") ~> institutionsRoutes ~> check {
         status mustBe StatusCodes.NotFound
+        responseAs[ErrorResponse] mustBe ErrorResponse(404, "2017 filing not found for institution xxxxx", "institutions/xxxxx/filings/2017")
```

Comment:
Is this the response we want? If a consumer of the api makes a request for a nonexistent institution, this response is misleading. I would think we want "Institution xxxxx not found".

I don't want to re-open this issue if it was already decided elsewhere, so let me know if there's discussion somewhere that I should refer to.